### PR TITLE
Second part

### PR DESCRIPTION
**Pull request by Olexandr-Sh**

### CODE REVIEW

File: css/_buttons.scss [hunk 1]
```diff
@@ -16,3 +16,58 @@
...
+  color: $prime-color;
+  border: 1px solid $prime-color;
+  outline: none;
+}
+.ratting-btn:active {
+  background-color: $second-color;
+  color: $bkg-tcolor-2;
+  border: 1px solid $bkg-tcolor-2;
+  outline: none;
+}
```
Comment (Plakhotenko) [inline css/_buttons.scss]: Need optimization. Common properties should be moved to common css rule.
`.ratting-btn:hover,
 .ratting-btn:focus,
 .ratting-btn:active {
   // common properties here
 }`

File: css/_buttons.scss [hunk 2]
```diff
@@ -16,3 +16,58 @@
...
+  color: $prime-color;
+  border: 1px solid $prime-color;
+  outline: none;
+}
+.other-options__btn:active {
+  background-color: $second-color;
+  color: $bkg-tcolor-2;
+  border: 1px solid $bkg-tcolor-2;
+  outline: none;
+}
```
Comment (Plakhotenko) [inline css/_buttons.scss]: the same

File: css/_hero-section.scss [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+  padding: 10px 0;
+}
+.ratting-btn {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.ratting-btn__users {
+  padding: 8px 8px 2px 8px;
```
Comment (Plakhotenko) [inline css/_hero-section.scss]:
```suggestion
  padding: 8px 8px 2px;
```

File: css/_buttons.scss (hunk 1 above)
Comment (Olexandr-Sh) [reply css/_buttons.scss]: Do you mean the outline properties?

Comment (Olexandr-Sh) [reply css/_buttons.scss]: I've customized focus effect of all buttons, so I decided to delete 'outline' in btns effects and added 'outline: none' to custom reset_scss file.

File: css/_buttons.scss [hunk 4]
```diff
@@ -7,12 +7,62 @@
...
+.ratting-btn:focus {
+  background-color: $bkg-tcolor-2;
+  color: $prime-color;
+  border: 1px solid $prime-color;
+}
+.ratting-btn:active {
+  background-color: $second-color;
+  color: $bkg-tcolor-2;
+  border: 1px solid $bkg-tcolor-2;
+}
```
Comment (Plakhotenko) [inline css/_buttons.scss:42]:
```suggestion
.ratting-btn:hover,
.ratting-btn:focus {
  background-color: $bkg-tcolor-2;
  color: $prime-color;
  border: 1px solid $prime-color;
}
.ratting-btn:active {
  background-color: $second-color;
  color: $bkg-tcolor-2;
  border: 1px solid $bkg-tcolor-2;
}

```

**Review by Plakhotenko — APPROVED** [review]

**Pull request merged**